controllers/result.go

```diff
@@ -29,15 +30,28 @@ func (c *ResultController) List() {
 
 func (c *ResultController) Create() {
 	c.EnsureAuthenticatedUser()
-
-	keywords := []string{
-		"ergonomic chair",
-		"cloud storage service",
-		"cloud computing service",
-		"crypto currency",
-		"เตา balmuda",
+	flash := web.NewFlash()
+
+	file, fileHeader, err := c.GetFile("file")
+	if err != nil {
+		flash.Error("Failed to upload file, please make sure the file is not corrupted")
```

Comment:
Error and success messages would be moved to a separate file later on [another task](https://github.com/carryall/go-google-scraper-challenge/issues/58)